modules/hydra-slave.nix

```diff
@@ -16,7 +16,7 @@ in
 
   environment.systemPackages = [ iohk-pkgs.iohk-ops ];
 
-  users.extraUsers.root.openssh.authorizedKeys.keys = pkgs.lib.singleton ''
-    command="nice -n20 ${pkgs.utillinux}/bin/flock -s /var/lock/lab nix-store --serve --write" ${pkgs.lib.readFile ./../static/id_buildfarm.pub}
-  '';
+  users.extraUsers.root.openssh.authorizedKeys.keys = map (key: ''
+    command="nice -n20 ${pkgs.utillinux}/bin/flock -s /var/lock/lab nix-store --serve --write" ${key}
```

Comment:
i think the flock part should go, its forcing the slave to only run 1 job at once, and hydra is configured to run 4 at once on each slave